src/components/headerArea.scss

```diff
@@ -0,0 +1,160 @@
+.header-area {
+  padding: var(--contentPaddingPrimary);
+}
+
+.top {
+  display: flex;
+  border-top: 1px solid #000;
```

Comment:
I know I keep harping on this, but we can probably set either this entire border style up as a variable, or at least use the existing `black` variable for the color.